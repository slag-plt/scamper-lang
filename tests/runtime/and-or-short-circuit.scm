(and (error "hello")
     #f)

(and #f
     (error "hello"))

(or (error "hello")
     #t)

(or #t
    (error "hello"))