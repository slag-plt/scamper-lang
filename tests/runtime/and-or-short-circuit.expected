:5:0: Runtime error:
    A runtime error was encountered: hello 
    In program: (error "hello")
#f
:4:6: Runtime error:
    A runtime error was encountered: hello 
    In program: (error "hello")
#t
